torch/_functorch/aot_autograd.py

```diff
@@ -1860,12 +1862,30 @@ def backward(ctx, *flat_args):
             )
             del contiguous_args
             if CompiledFunction.compiled_bw is None:
-                # TODO - pass in fake tensors ?
-                context = disable_autocast_manager if disable_amp else nullcontext
-                with context(), track_graph_compiling(aot_config, "backward"):
-                    CompiledFunction.compiled_bw = aot_config.bw_compiler(
-                        bw_module, all_args
-                    )
+                all_args_list = list(all_args)
+                CompiledFunction.compiled_bw = create_aot_dispatcher_function(bw_module, all_args_list, AOTConfig(aot_config.bw_compiler, None, None, aot_config.decompositions, 0, aot_config.aot_id))
```

Comment:
LOL

Reply:
This is probably what's causing all the failures :think:

Reply:
In particular, if you do it this way, I don't think you end up passing any real tensors to the backend (which means, e.g., cuda graphs doesn't work)